components/usage/pkg/apiv1/billing.go

```diff
@@ -165,6 +167,76 @@ func (s *BillingService) CreateStripeCustomer(ctx context.Context, req *v1.Creat
 	}, nil
 }
 
+func (s *BillingService) CreateStripeSubscription(ctx context.Context, req *v1.CreateStripeSubscriptionRequest) (*v1.CreateStripeSubscriptionResponse, error) {
+	attributionID, err := db.ParseAttributionID(req.GetAttributionId())
+	if err != nil {
+		return nil, status.Errorf(codes.InvalidArgument, "Invalid attribution ID %s", attributionID)
+	}
+
+	customer, err := s.GetStripeCustomer(ctx, &v1.GetStripeCustomerRequest{Identifier: &v1.GetStripeCustomerRequest_AttributionId{AttributionId: string(attributionID)}})
```

Comment:
```suggestion
	customer, err := s.GetStripeCustomer(ctx, &v1.GetStripeCustomerRequest{
		Identifier: &v1.GetStripeCustomerRequest_AttributionId{
			AttributionId: string(attributionID),
		},
	})
```
Just makes it a bit more readable, especially on a smaller screen.